bugbug/bug_features.py

```diff
@@ -134,6 +134,10 @@ def cleanup_fileref(text):
     return re.sub(r'\w+\.py\b|\w+\.json\b|\w+\.js\b|\w+\.jsm\b|\w+\.html\b|\w+\.css\b|\w+\.c\b|\w+\.cpp\b|\w+\.h\b', '__FILE_REFERENCE__', text)
 
 
+def cleanup_dll(text):
+    return re.sub(r'\w+\.dll |\w+\.dll@\w+', '__DLL_NAME__ ', text)
```

Comment:
A word doesn't always finish with a space, but it could finish with other characters (e.g. a comma). You can use `\b` to define a word boundary.
Also, it would be better to have the common part at the beginning and then the variable part in brackets at the end.
E.g. if I want to match file1 and file2, I'd write file[12] and not file1|file2.